src/server/database/Database/Transaction.cpp

```diff
@@ -63,20 +67,51 @@ void Transaction::Cleanup()
     _cleanedUp = true;
 }
 
+const char* Transaction::ToString()
+{
+    std::string msg;
+    for (SQLElementData const& data : m_queries)
+    {
+        switch (data.type)
+        {
+        case SQL_ELEMENT_PREPARED:
+            msg += data.element.stmt->getQueryString() + ';';
+            break;
+        case SQL_ELEMENT_RAW:
+            msg += data.element.query + ';';
```

Comment:
From travis-ci :
```cpp
src/server/database/Database/Transaction.cpp:81:39:
 fatal error: adding 'char' to a string pointer does not append to the string [-Wstring-plus-char]
            msg += data.element.query + ';';
                   ~~~~~~~~~~~~~~~~~~~^~~~~
 note: use array indexing to silence this warning
            msg += data.element.query + ';';
                                      ^
                   &                  [    ]
```
(this warning is from debug mode, so Travis treats it as "fatal error")

Reply:
So exciting. ^__^ I'll make those changes and release very soon.